AppCenterCrashes/AppCenterCrashes.xcodeproj/project.pbxproj

```diff
@@ -787,7 +787,7 @@
 		049327981ECA19FD00D0187A /* PLCrashReporter */ = {
 			isa = PBXGroup;
 			children = (
-				DFEB97F72318F42700AE88C1 /* CrashReporter */,
+				DFEB97F72318F42700AE88C1 /* include */,
```

Comment:
Why is this change? Why it's now in `include` vs in ``CrashReporter` before?

Reply:
It was to emulate "frameworks" imports `<CrashReporter/CrashReporter.h>`. It's just regular headers with static lib now.
Framework headers are handled in a special way by compiler, so we shouldn't try to emulate this.
It had to be done when it's replaced from frameworks